datasets/ua/war_sanctions/crawler.py

```diff
@@ -0,0 +1,348 @@
+from zavod import Context, helpers as h
+
+
+LINKS = [
+    {  # child kidnappers
+        "url": "https://war-sanctions.gur.gov.ua/en/kidnappers/persons?page=1&per-page=12",
+        "type": "person",
+    },
```

Comment:
Let's add something like `List of persons involved in the deportation of Ukranian children` (sourced from the homepage) as `Sanction:program`

Same for the other lists.